install.sh

```diff
@@ -240,9 +240,9 @@ There is another binary in your PATH that conflicts with the binary we've instal
 
   $1
 
-This is likely because of an existing installation of Coder. See our documentation for suggestions on how to resolve this.
-
-  https://coder.com/docs/v2/latest/install/install.sh#path-conflicts
+This is likely because of an existing installation of Coder resulting in multiple Coder binaries in your PATH.
+You can check where all of the versions are located by running \`which -a coder\`.
+Whichever binary comes first in the returned list will be used when running Coder commands.
```

Comment:
I trimmed up the language a bit, let me know your thoughts!

Could you manually confirm that `$PATH` is properly escaped here if accepted?

```suggestion
This is likely because of an existing installation of Coder in your \$PATH.

Run `which -a coder` to view all installations.
```

Reply:
Probably not necessary, but:
> Run `which -a coder` to view all installations.
to:
> Run `which -a coder` to view all installations (first come, first served).

Reply:
I like Kyle's version as is personally